src/main/resources/account/getAccountSettings.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<template xmlns="http://ws.apache.org/ns/synapse" name="getAccountSettings">
+    <parameter name="apiVersionGetAccountSettings" description="API version used for GetAccountSettings method."/>
+    <sequence>
+        <property name="uri.var.apiVersion" expression="$func:apiVersionGetAccountSettings"/>
+        <property name="uri.var.uriRemainder" expression="fn:concat('/', $ctx:uri.var.apiVersion,
+         '/account-settings/')"/>
+        <property name="httpMethod" value="GET"/>
+        <class name="org.wso2.carbon.esb.connector.auth.AmazonLambdaAuthConnector"/>
+        <header name="Authorization" scope="transport" expression="$ctx:authorizationHeader"/>
+        <header name="X-Amz-Date" scope="transport" expression="$ctx:x-amz-date"/>
+        <payloadFactory media-type="json">
+            <format/>
+            <args/>
+        </payloadFactory>
```

Comment:
Do we need this payloadFactory mediator?

Reply:
Fix with 408c7bd